src/logic/safe/api/fetchTokenCurrenciesBalances.ts

```diff
@@ -95,12 +117,16 @@ const tokensQuery = `query { tokens(number_gte: 11749000) {
 }}`
 
 async function fetchPrices(): Promise<Record<string, number>> {
-  const { tokens } = await request<{ tokens: PriceInfo[] }>(
-    'https://api.thegraph.com/subgraphs/name/ubeswap/ubeswap',
-    tokensQuery,
-  )
-  return tokens.reduce((collection, entry) => {
-    collection[entry.symbol.toUpperCase()] = entry.derivedCUSD
-    return collection
-  }, {})
+  try {
+    const { tokens } = await request<{ tokens: PriceInfo[] }>(
+      'https://api.thegraph.com/subgraphs/name/ubeswap/ubeswap',
+      tokensQuery,
+    )
+    return tokens.reduce((collection, entry) => {
+      collection[entry.symbol.toUpperCase()] = entry.derivedCUSD
+      return collection
+    }, {})
+  } catch {
+    return {}
```

Comment:
We're sorta swallowing this error now and I'm wondering if we could do something with it.